guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -411,51 +411,44 @@
+    "#### For ArcGIS Pro users:"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "For opening Python Command Prompt navigate to  \n",
-    "**Start** Menu &gt; **All Programs** &gt; **ArcGIS** &gt; **Python Command Prompt**\n",
-    "\n",
-    "If using Anaconda, Open a terminal application, navigate to the directory you want to work in and activate the conda environment.\n",
-    "\n",
+    "To install deep learning packages in ArcGIS Pro, first ensure that ArcGIS Pro is installed.\n",
     "\n",
-    "Next, you will install the following Python packages: Tensorflow, fast.ai, Keras, Pytorch, Scikit-image, Pillow, and Libtiff. Keep in\n",
-    "mind you will be installing specific versions of these packages indicated in the installation steps below. The tools only work with\n",
-    "these specific versions. \n",
+    "[Deep learning frameworks](https://github.com/esri/deep-learning-frameworks) can be used to install all the required dependencies in ArcGIS Pro's default python environment using an MSI installer.\n",
     "\n",
+    "Or alternatively,\n",
     "\n",
-    "- Install the dependencies using the following command:\n",
+    "for a cloned environment of ArcGIS Pro's default environment, `deep-learning-essentials` metapackage can be used to install the required dependencies. Which can be done using the following command, in the `Python Command Prompt` *(included with ArcGIS Pro)*:\n",
     "\n",
-    "    ```conda install -c esri -c fastai -c pytorch arcgis=1.8.2 scikit-image=0.15.0 pillow=6.2.2 libtiff=4.0.10 fastai=1.0.60 pytorch=1.4.0 torchvision=0.5.0 --no-pin```\n"
+    "```conda install -c esri deep-learning-essentials```\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- For TensorFlow support (optional), use the following command:\n",
-    "\n",
-    "    ```conda install -c esri -c fastai -c pytorch arcgis=1.8.2 scikit-image=0.15.0 pillow=6.2.2 libtiff=4.0.10 fastai=1.0.60 pytorch=1.4.0 torchvision=0.5.0 tensorflow-gpu=2.1.0 --no-pin```"
+    "####  For Anaconda users (Windows & Linux platforms):"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- For Multispectral data support (optional and needed only for Anaconda users)\n",
+    "`arcgis_learn` metapackage can be used for both windows and linux installations of Anaconda in a new environment.\n",
+    "\n",
+    "The following command will update Anaconda to the latest version.\n",
+    "\n",
+    "```conda update conda```\n",
+    "\n",
+    "After that, metapackage can be installed using the command below:\n",
     "\n",
-    "   ```conda install -c esri gdal=2.3.3```\n",
-    "  \n",
-    "  \n",
-    "  \n",
-    "Alternatively, you may also use [deep learning essentials](https://anaconda.org/esri/deep-learning-essentials) to install deep learning environment."
+    "```conda install -c esri arcgis_learn python=3.7```"
```

Comment:
`conda install -c esri arcgis_learn python=3.7`
also as priyanka pointed out,
we can add text info about the fact that we also support 3.6 and 3.8 python version.
but in terms of command, we can keep that as for python 3.7 as Pro uses python 3.7 .

side note: python version is added in the command as, without version, it can take some time, 10-15 min.  or plus in resolving dependencies and auto-selecting python version.
but if the python version is mentioned then dependencies are resolved within few seconds.